### DISCUSSION TITLE
How do you model non networking devices in NetBox?

### DISCUSSION THREAD

**Discussion by julianstolp** [netbox-community/netbox#15921] [Q&A] [answered]
I was wondering how to integrate various devices (monitors, docking stations and other technical devices) into NetBox. Of course, the devices have to be created as ‘devices’, but I am wondering how best to integrate them logically.

**Comment by candlerb** [comment, marked as answer]
Netbox isn't really an asset management system. Nothing stops you creating those items as unracked devices, but this isn't what it's designed to do.

Other more asset-focussed projects I have heard of which you might want to check out include [GLPI](http://www.glpi-project.org/), [SnipeIT](https://snipeitapp.com/), [InvenTree](https://inventree.org/), and [REI3](https://rei3.de/en/applications/it_assets).
